pixeltable/catalog/table_version.py

```diff
@@ -495,6 +495,7 @@ def drop_index(self, idx_id: int) -> None:
         assert idx_md.name in self.idxs_by_name
         idx_info = self.idxs_by_name[idx_md.name]
         del self.idxs_by_name[idx_md.name]
+        del self.idx_md[idx_id]
```

Comment:
missing explanatory comment

Reply:
I have added a comment, please let me know if this is what you had in mind.
<!-- Reviewable comment -OEBdXhl-9QQpr0JCut_:bxw2l7w -->
<!-- Sent from Reviewable.io -->